src/ORM/Connect/Database.php

```diff
@@ -229,6 +260,28 @@ protected function benchmarkQuery($sql, $callback, $parameters = array())
         }
     }
 
+    /**
+     * Display query message
+     *
+     * @param mixed $query
+     * @param float $endtime
+     */
+    private function displayQuery($query, $endtime)
```

Comment:
Please use protected

Reply:
@robbieaverill Is it the matter of consistency again?

Reply:
Yep, we prefer protected over private in SilverStripe development :-)